.github/workflows/ci.yml

```
@@ -9,17 +9,28 @@ on:
       - master
 
 jobs:
-  test-ubuntu:
+  test-debian:
     strategy:
       matrix:
-        ubuntu:
-          - 16.04
-          - latest
-    runs-on: ubuntu-${{ matrix.ubuntu }}
+        image:
+          - ubuntu:latest
+          - ubuntu:16.04
+          - debian:testing
+          - debian:stable
+          - debian:oldstable
+    runs-on: ubuntu-latest
+    container:
+      image: docker.io/library/${{ matrix.image }}
```

```suggestion
      image: ${{ matrix.image }}
```

This should be enough. This would also allow us to use images from different registries if we wanted.

---

Done.